ebusd-2.1.x/de/vaillant/08.bai.csv

```diff
@@ -13,7 +13,7 @@
 [PROD='0010006341']!load,bai.0010006341.inc,,,
 [PROD='0010007508';'0010007510';'0010007512';'0010007514';'0010007516';'0010007518';'0010007520';'0010007522';'0010007524';'0010007526';'0010007688';'0010007692';'0010007696';'0010007700';'0010007704']!load,bai.0010007508.inc,,,
 [PROD='0010010674';'0010010676';'0010010678';'207']!load,bai.0010010674.inc,,,
-[PROD='0010014629';'0010014630';'0010014631';'0010014632';'0010014633';'0010015596';'0010015597';'0010015598';'0010015599';'0010015600';'0010015601';'0010015602';'0010015603';'0010015604';'0010015605';'0010015606';'0010015607';'0010015608';'0010015609';'0010015610';'0010015611';'0010015612';'0010015613';'0010015614';'0010019268';'0010021888']!load,bai.0010015600.inc,,,
+[PROD='0010014917';PROD='0010014629';'0010014630';'0010014631';'0010014632';'0010014633';'0010015596';'0010015597';'0010015598';'0010015599';'0010015600';'0010015601';'0010015602';'0010015603';'0010015604';'0010015605';'0010015606';'0010015607';'0010015608';'0010015609';'0010015610';'0010015611';'0010015612';'0010015613';'0010015614';'0010019268';'0010021888']!load,bai.0010015600.inc,,,
```

Comment:
Is the double PROD needed?

Reply:
Thank you for your check. I will correct it.